its/install.ts

```diff
@@ -0,0 +1,54 @@
+import { DefinitionQueryOrder } from "azure-devops-node-api/interfaces/BuildInterfaces";
+import fs from "fs";
+import path from "path";
+import { AZDO_PIPELINE_NAME_PREFIX, AZDO_PROJECT, FIXTURES_PATH } from "./constant";
+import { createPipeline, getAzdoApi, getTemplatePipeline } from "./it/azdo";
+
+const CLEANUP_PAGINATION_LIMIT = 500;
+
+async function main() {
+  const azdoApi = getAzdoApi();
+  const buildApi = await azdoApi.getBuildApi();
+
+  // List all files in the fixtures directory
+  const pipelineNames = fs
+    .readdirSync(FIXTURES_PATH)
+    .filter((file) => {
+      return file.endsWith(".yml") && file.startsWith(AZDO_PIPELINE_NAME_PREFIX);
+    })
+    .map((filePath) => filePath.replace(".yml", ""));
```

Comment:
is there a relationship between the release pipeline and the CI check to ensure that all pipelines are freshly generated?

Reply:
No relation, I'll add a task in the release pipeline to check that the generated pipelines are up to date

Reply:
I don't want to over-engineer; I was thinking we would fail the build in this case
`npm run generate-its && git diff --exit-code # Ensure that the generated ITS are up-to-date`
but I realize that is running in cirrus, and this is all in azdo so it wouldn't be blocked 😢

Reply:
I've added a check on Azure as well
![image](https://github.com/user-attachments/assets/00ce646c-0a09-4eae-863a-86239c630638)